### ISSUE TITLE
BUG: no internet tun

### ISSUE THREAD

**Issue by harneizel** [MatsuriDayo/nekoray#1477]
I am using windows 10, and when I start nekoray with tun mode enabled, the Internet disappears on my PC. I tried changing DNS configurations, but it didn't help. everything works on the phone.
![IMG_20241027_231903](https://github.com/user-attachments/assets/843d4299-a34f-4a49-9d83-ab6ceb7f65eb)

**Comment by laniakeadev** [comment]
Same 
UPD: fixxed by filling right SNI (server names) that's not blocked

**Comment by tacchyon69** [comment]
Yeah same bro was using it fine without issues till recently now its just doesnt work I dont know what happened i hope someone answer this

**Comment by laniakeadev** [comment]
> Yeah same bro was using it fine without issues till recently now its just doesnt work I dont know what happened i hope someone answer this

Check out that your SNI is available

**Comment by tacchyon69** [comment]
> > Yeah same bro was using it fine without issues till recently now its just doesnt work I dont know what happened i hope someone answer this
> 
> Check out that your SNI is available

I managed to fix it by setting setting time automaticaly in windows it was off for some reason and changing direct dns in routing settings to https://223.5.5.5/dns-query if i put local host sing-tun doesnt have internet for some fucking reason 
no more inhaling bochium tho
